tests/operator_tests.py

```diff
@@ -280,6 +280,51 @@ def testConcatenate(self):
             (ak.array([True, False, True, False, True, True]) == ak.concatenate([pdaOne, pdaTwo])).all()
         )
 
+    def test_float_uint_binops(self):
+        # Test fix for issue #1620
+        ak_uint = ak.array([5], dtype=ak.uint64)
+        np_uint = np.array([5], dtype=np.uint64)
+        scalar_uint = np.uint64(5)
+
+        ak_float = ak.array([3.01], dtype=ak.float64)
+        np_float = np.array([3.01], dtype=np.float_)
+        scalar_float = 3.01
+
+        ak_uints = [ak_uint, scalar_uint]
+        np_uints = [np_uint, scalar_uint]
+        ak_floats = [ak_float, scalar_float]
+        np_floats = [np_float, scalar_float]
+        for aku, akf, npu, npf in zip(ak_uints, ak_floats, np_uints, np_floats):
```

Comment:
looking back at this, i realize this is only testing `ak_uint` against `ak_float` and never comparing either against the scalar versions... This can be solved by a nested `for` but their might be a better way, either way this needs to be updated

Reply:
actually im wrong, ignore me.... this code is fine lmao